### ISSUE TITLE
More complete classic/Platinum appearance

### ISSUE THREAD

**Issue by mihaip** [mihaip/infinite-mac#162]
`<select>`s should have an era-appropriate look too.

The buttons should have focus rings.

See also https://sakofchit.github.io/system.css/

**Comment by that-ben** [comment]
"sakofchit" 🤣

**Comment by mihaip** [comment]
af28b201cb181244865381007a0cf5b88de095f4 and 330fa7dda0503f12ff9c67bd05b9365b5995092b did some of this, but we're still missing button focus rings and the era-appropriate fonts.
